client/styles/welcome.scss

```diff
@@ -4,25 +4,20 @@
   padding: 0;
```

Comment:
![image](https://user-images.githubusercontent.com/23039408/156717592-136d1a11-326d-45c9-937f-accb16d6d146.png)
Login page looks off - can you fix it to use the layout?